modules/dashboard/src/App.js

```diff
@@ -54,7 +54,7 @@ const env = {
   urlPrefix: process.env.PUBLIC_URL || "",
 };
 
-// env.nodeUrl = "wss://indra.connext.network/api/messaging";
+env.nodeUrl = "wss://staging.indra.connext.network/api/messaging";
```

Comment:
is there a reason this is hardcoding this instead of using the overrides?

Reply:
makes my life easier locally, just forgot to comment it before pushing